.eslintrc.js

```diff
@@ -16,30 +14,36 @@ module.exports = {
     node: true,
   },
   rules: {
-    "semi": ["error", "always"],
```

Comment:
Note, the rest of this particular file is all formatting (i.e., no functional changes)